website/src/components/datasources-list-section.tsx

```diff
@@ -0,0 +1,95 @@
+import { Fragment } from 'react';
+import { cn, DecorationIsolation, Heading, MeshIcon } from '@theguild/components';
+
+const datasources = [
+  {
+    name: 'GraphQL',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/graphql',
+  },
+  {
+    name: 'OpenAPI',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/openapi',
+  },
+  {
+    name: 'gRPC',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/grpc',
+  },
+  {
+    name: 'JSON Schema',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/json-schema',
+  },
+  {
+    name: 'SOAP',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/soap',
+  },
+  {
+    name: 'OData',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/odata',
+  },
+  {
+    name: 'Thrift',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/thrift',
+  },
+  {
+    name: 'SQLite',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/sqlite',
+  },
+  {
+    name: 'MySQL',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/mysql',
+  },
+  {
+    name: 'Neo4j',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/neo4j',
+  },
+  {
+    name: 'PostgreSQL',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/postgres',
+  },
+  {
+    name: 'MongoDB',
+    href: 'https://the-guild.dev/graphql/mesh/v1/source-handlers/mongodb',
+  },
+];
+
+export function DatasourcesListSection(props: React.HTMLAttributes<HTMLDivElement>) {
+  return (
+    <section
+      {...props}
+      className={cn(
+        'bg-beige-100 relative rounded-3xl text-green-1000 p-4 pt-6 sm:py-24 md:px-6 lg:px-12 2xl:px-24 md:py-[120px]',
+        props.className,
+      )}
+    >
+      <DecorationIsolation className="absolute z-10">
+        <MeshIcon
+          width={871}
+          height={865}
+          className="[&>g]:fill-none stroke-beige-300 [&>g]:stroke-[0.1px] [&>g]:[clip-path:none] right-0 bottom-0 absolute translate-x-1/2 translate-y-1/2"
```

Comment:
```suggestion
          className="[&>g]:fill-none stroke-beige-300 [&>g]:stroke-[0.1px] [&>g]:[clip-path:none] right-0 bottom-0 absolute translate-1/2"
```

Reply:
I'm not sure `translate-{}` works here.
https://tailwindcss.com/docs/translate

Reply:
yep, sorry